src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * This ByeCommand class will print the exit message when executed.
+ */
+public class ByeCommand extends Command {
+
+    public static final String COMMAND_WORD = "bye";
+
+    /**
+     * Executes command by printing exit message.
+     * @param taskList List of tasks.
```

Comment:
Should there be a line break between the description and the block tags here? I have also noticed other places where this is omitted.